src/spec.ts

```diff
@@ -606,9 +604,9 @@ function normalizePathOverlay(spec: NormalizedUnitSpec, config: Config = {}): No
     layer.push({
       ...(projection ? {projection} : {}),
       mark: {
-        type: 'point',
         opacity: 1,
         filled: true,
+        type: 'point',
```

Comment:
Not sure why you have to change the order here. I think putting mark type at the top is more desirable.

Reply:
Consistency with the other one. If we want to copy some properties from the main mark def, we want to do it after opacity and filled.